packages/ipfs-http-client/src/lib/ipld-formats.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+const dagPB = require('ipld-dag-pb')
+const dagCBOR = require('ipld-dag-cbor')
+const raw = require('ipld-raw')
+const multicodec = require('multicodec')
+
+/**
+ * @typedef {import('cids')} CID
+ */
+
+/**
+ * Return an object containing supported IPLD Formats
+ *
+ * @param {object} [options] - Options passed to the http client constructor
+ * @param {object} [options.ipld] - IPLD options passed to the http client constructor
+ * @param {Array} [options.ipld.formats] - A list of IPLD Formats to use
+ * @param {Function} [options.ipld.loadFormat] - An async function that can load a format when passed a codec name
+ * @returns {Function}
+ */
+module.exports = (options) => {
```

Comment:
```suggestion
module.exports = (options = {}) => {
```

We are checking if options exists several times below, can we just set the default value here instead?